### PR TITLE
Add extensive documentation on how to set up a machine to flash sensors.

### PR DESCRIPTION
**Pull request by JHGZhou**
**do it**

### CODE REVIEW

**Review by yizow — CHANGES_REQUESTED** [review]
good

**Review by baby-bell — CHANGES_REQUESTED** [review]
- A couple typos: pour->poor, cahinge->change.
- Consider just directly putting in a link.
- There are device names in the Travis Makefile.

**Comment by JHGZhou** [comment]
@ebronstein I've made the fixes.

**Comment by JHGZhou** [comment]
@baby-bell I've made the edits.

File: hibike/README.md [hunk 1]
```diff
@@ -430,12 +430,12 @@ Error handling
...
 
 Then, run the setup script in hibike. `sudo ./setup.sh` in `piecentral/hibike` should do the trick. It will download a lot of stuff. You have effectively given up all your software freedoms at this point. Go cry a bit. Come back after asking Linus for forgiveness. 
 
-You're almost done, keep it up. Edit `/usr/bin/ard-reset-arduino`. You should sudo this. Don't worry, this is the last time you'll be violating your system. cahinge `ser.setBaudrate(1200)` to `se.baudrate = 1200`. Now your pour machine is ready to flash Arduinos (tm)
+You're almost done, keep it up. Edit `/usr/bin/ard-reset-arduino`. You should sudo this. Don't worry, this is the last time you'll be violating your system. Change `ser.setBaudrate(1200)` to `se.baudrate = 1200`. Now your poor machine is ready to flash Arduinos (tm)
 
-Flash by going to `piecentral/hibike`. The command to run is `./flash.sh SENSOR_NAME`. Figure out what the sensor names are I don't freaking know.   
+Flash by going to `piecentral/hibike`. The command to run is `./flash.sh SENSOR_NAME`. Figure out what the sensor names are in the Travis Makefile.
```
Comment (vivienngyn) [inline hibike/README.md]: Please just list the sensor names for ease of use:

The command to run is `./flash.sh SENSOR_NAME`, where `SENSOR_NAME` is one of 

- `BatteryBuzzer`
- `ExampleDevice`
- `LimitSwitch`
- `LineFollower`
- `Potentiometer`
- `RFID`
- `ServoControl`
- `TeamFlag`
- `YogiBear`

**Review by viviehn — CHANGES_REQUESTED** [review]

**Review by baby-bell — APPROVED** [review]
LGTM aside from Vivien's comment

**Comment by onibrow** [comment]
Closed.

**Pull request closed without merging**